help/md/fnTrN.md

```diff
@@ -1,19 +1,45 @@
 ## name
 fnTrN
+
 ## title
 The Tamura-Nei (1993) nucleotide rate matrix
+
 ## description
-The Tamura-Nei nucleotide rate matrix.
+DNA evolution model proposed in Tamura & Nei (1993).
+
 ## details
+In this model, nucleotide base frequencies are different, and the two transition rates (A <-> G and C<->T) can be different to each other, and to the transversion rate. The first argument, kappa1, defines the ratio between the rate of A <-> G (i.e. purine) transitions to transversions. The second argument, kappa2, defines the ratio between the rate of C <-> T (i.e. pyrimidine) transitions to transversions. The third argument, baseFrequencies, defines the stationary frequencies of nucleotide bases. 
+
+The TrN rate matrix elements are of the form:
+	Q[i, j] = c * kappa1 * baseFrequencies[j], if i<->j is A<->G
+			= c * kappa2 * baseFrequencies[j], if i<->j is C<->T
+			= c * baseFrequencies[j], otherwise
+
+where c is a constant needed to normalize the average rate to 1 
+
 ## authors
 ## see_also
+fnJC
+fnK80
+fnK81
+fnT92
 fnHKY
+fnGTR
+
 ## example
-        kappaAG ~ dnLognormal(0,1)    # The purine transition rate
-        kappaCT ~ dnLognormal(0,1)    # The pyrimindine transition rate
-        pi ~ dnDirichlet( v(1,1,1,1) )
-        Q := fnTrN(kappaAT, kappaCT, ,pi)
+	# A <-> G transition rate
+	kappaAG ~ dnLognormal(0,1)
+
+	# C <-> T transition rate
+	kappaCT ~ dnLognormal(0,1)
+
+	# nucleotide base frequencies
+	pi ~ dnDirichlet( v(1,1,1,1) )
+
+	# create a TrN rate matrix
+	Q := fnTrN(kappaAT, kappaCT, ,pi)
```

Comment:
`kappaAT` is a typo -- should be `kappaAG` I think

Reply:
Thanks! Fixed